Mage/src/main/java/mage/watchers/common/SourceDidDamageWatcher.java

```diff
@@ -24,11 +24,9 @@ public SourceDidDamageWatcher() {
 
     @Override
     public void watch(GameEvent event, Game game) {
-        if (event.getType() == GameEvent.EventType.DAMAGED_CREATURE
-                || event.getType() == GameEvent.EventType.DAMAGED_PLANESWALKER
+        if (event.getType() == GameEvent.EventType.DAMAGED_PERMANENT
                 || event.getType() == GameEvent.EventType.DAMAGED_PLAYER) {
```

Comment:
I recommends to change `DAMAGED_PERMANENT`+ `DAMAGED_PLAYER` to `instanceOf` damage event, cause it look at any damage.

Reply:
yeah I thought about that but it was so much refactoring that I just wanted to change it as quickly as possible